src/flavors/mississippi/config.json

```diff
@@ -0,0 +1,799 @@
+            "type": "symbol",
+            "source": "mississippi-input",
+            "layout": {
+              "icon-size": 0.4,
+              "icon-anchor": "bottom",
+              "icon-image": [
+                "match",
+                ["get", "location_type"],
+                "geology",
+                "geology.png",
+                "history",
+                "history.png",
+                "industry",
+                "industry.png",
+                "culture",
+                "culture.png",
+                "ecology",
+                "ecology.png",
+                "__no-icon-image__"
+              ]
+            }
+          }
+        ],
+        "mapboxLayers": [
+          {
+            "id": "mississippi-input",
+            "type": "symbol",
+            "source": "mississippi-input",
+            "layout": {
+              "icon-allow-overlap": true,
+              "icon-anchor": "bottom",
+              "icon-size": ["step", ["zoom"], 0.2, 10, 0.3],
+              "icon-image": [
+                "step",
+                ["zoom"],
+                [
+                  "match",
+                  ["get", "location_type"],
+                  "geology",
+                  "geology.png",
+                  "history",
+                  "history.png",
+                  "industry",
+                  "industry.png",
+                  "culture",
+                  "culture.png",
+                  "ecology",
+                  "ecology.png",
+                  "__no-icon-image__"
+                ],
+                10,
+                [
+                  "match",
+                  ["get", "location_type"],
+                  "geology",
+                  "geology.png",
+                  "history",
+                  "history.png",
+                  "industry",
+                  "industry.png",
+                  "culture",
+                  "culture.png",
+                  "ecology",
+                  "ecology.png",
+                  "__no-icon-image__"
+                ]
+              ]
+            }
+          }
+        ]
+      },
+      {
+        "id": "nitrogen-group",
+        "basemap": false,
+        "visibleDefault": false,
+        "mapboxLayers": [
+          {
+            "id": "nitrogen-layer",
+            "type": "raster",
+            "source": "nitrogen"
+          }
+        ]
+      },
+      {
+        "id": "nitrogen-plants-group",
+        "basemap": false,
+        "visibleDefault": false,
+        "interactive": true,
+        "popupContent": "<p><strong>test!</strong></p>",
+        "mapboxLayers": [
+          {
+            "id": "nitrogen-plants-labels",
+            "type": "circle",
+            "source": "nitrogen-plants",
+            "source-layer": "nitrogen-plants",
+            "paint": {
+              "circle-radius": {
+                "base": 1.75,
+                "stops": [[12, 2], [22, 180]]
+              },
+              "circle-color": "#fbb03b"
```

Comment:
I'm trying to draw this geojson source as circles, but they don't appear on the map. @goldpbear any suggestions on how to debug this?

Reply:
I think removing the `source-layer` line at line 259 will fix this. `source-layer` is only needed for layers rendering vector tile sources.

In general, Mapbox logs pretty good debugging information. There are cases, though, (like here) where Mapbox seems to swallow misconfigured layers and just won't render them. I've worked through a few situations like that so I might be able to help debug layers that seemingly won't render.

Reply:
Haha, yes, that did the trick!

Thanks for the tip :+1: